app/views/adoption_requests/new.html.erb

```diff
@@ -1,5 +1,5 @@
 <% provide(:title, "Aditional Information for Adoption Request") %>
-<h1><%= @adoption_request.gems_adoption %></h1>
+<h1><%= @adoption_request.gems_adoption.gem %></h1>
```

Comment:
It seems like this change is causing a test to error, check Travis: https://travis-ci.org/rubygems/adoption-center/jobs/76410806.